qa/automation/CareerPortal.Tests.Scenarios/3. Buttons/3.1. Buttons action.feature

```diff
@@ -0,0 +1,64 @@
+Feature: 3.1 Actions
```

Comment:
For me it isn't obvious what does 'Actions' mean. All test are about buttons behaviour, maybe it will be more clear to name it 'Buttons behaviour' or something like that?